r2/r2/controllers/api.py

```diff
@@ -795,6 +800,86 @@ def check_api_friend_oauth_scope(self, type_):
                 # this request.
                 self._auth_error(403, "insufficient_scope")
 
+    def _handle_mod_successorship(self, sr):
```

Comment:
You may want these cycles being just one, the list to be cycled through determined via `getattr`